### ISSUE TITLE
Bybit `fetchMyLiquidations` failed due to safeMarket() 

### ISSUE THREAD

**Issue by marcustut** [ccxt/ccxt#23276]
### Operating System

_Linux_

### Programming Languages

_Nodejs_

### CCXT Version

_4.3.69_

### Description

failed to fetch due to ArgumentsRequired: bybit safeMarket() requires a fourth argument for BTCUSDT to disambiguate between different markets with the same market id, will fetch again in 1000ms

### Code

```ts
const exchange = new ccxt.bybit({ apiKey, secret });
client['options']['defaultType'] = 'swap';
await exchange.fetchMyLiquidations();
```

**Comment by carlosmiei** [comment]
Hello @marcustut, thanks for reporting it, we will take a look asap.

**Comment by marcustut** [comment]
> Hello @marcustut, thanks for reporting it, we will take a look asap.

sorry I accidentally submitted an empty issue, I just updated the issue with more info. Thanks for the swift reply! It is due to in `bybit.ts`'s `parseLiquidation`, there's a `this.safeSymbol` call and that called `this.safeMarket`  which triggered this

**Comment by carlosmiei** [comment]
@marcustut Can you please upgrade your ccxt version and try again? We released a new version (4.3.70) with the fix.

**Comment by marcustut** [comment]
> @marcustut Can you please upgrade your ccxt version and try again? We released a new version (4.3.70) with the fix.

I upgraded and can confirm that the fix indeed worked, thanks!
